packages/grafana-data/src/field/fieldState.ts

```diff
@@ -1,10 +1,18 @@
-import { DataFrame, Field, TIME_SERIES_VALUE_FIELD_NAME, FieldType, TIME_SERIES_TIME_FIELD_NAME } from '../types';
+import { getFieldMatcher } from '../transformations';
+import {
+  DataFrame,
+  Field,
+  TIME_SERIES_VALUE_FIELD_NAME,
+  FieldType,
+  TIME_SERIES_TIME_FIELD_NAME,
+  FieldConfigSource,
+} from '../types';
 import { formatLabels } from '../utils/labels';
 
 /**
  * Get an appropriate display title
  */
-export function getFrameDisplayName(frame: DataFrame, index?: number) {
+export function getFrameDisplayName(frame: DataFrame, index?: number, singleLabelName?: string) {
```

Comment:
we may want to avoid modifying this function as part of these changes until we can verify we aren't introducing any regressions

Reply:
already reverted